configs/modify-vm-template.yaml

```diff
@@ -0,0 +1,4 @@
+task_name: modify-vm-template
+task_category: modify-vm-template
+main_image: quay.io/kubevirt/tekton-modify-vm-template:v0.0.1
```

Comment:
I think this should have a `tekton-task` prefix as the other tasks. 

Are you using the onboard-new-task script?